tools/c7n_azure/tests/azure_common.py

```diff
@@ -75,6 +75,10 @@ def request_callback(self, request):
                 r"[\da-zA-Z]{8}-([\da-zA-Z]{4}-){3}[\da-zA-Z]{12}",
                 DEFAULT_SUBSCRIPTION_ID,
                 request.url)
+        if request.body:
+            request.body = b'mock_body'
```

Comment:
Is it really OK to wipe out the whole body?  I guess if the tests pass for now it must be OK :) If it isn't OK for some other API someone can change it then.

Reply:
OIC - no one would ever see them from the cassette I guess? I think I get it now!

Reply:
Haha yeah because we only check the request.uri - we never do anything with the body! At least my quick scan noticed. All tests pass so I figured this was easiest since there were SO many keys and identifiers in the body :P